packages/nextra/src/loader.ts

```diff
@@ -100,10 +100,11 @@ async function loader(
   const mdxPath = (
     context._module?.resourceResolveData
       ? // to make it work with symlinks, resolve the mdx path based on the relative path
-        path.join(
-          context.rootContext,
-          context._module.resourceResolveData.relativePath
-        )
+        /*
+         * `context._module.resourceResolveData.relativePath` could include path chunk of
+         * `context.rootContext` use `context._module.rawRequest` instead
+         */
+        path.join(context.rootContext, context._module.rawRequest)
```

Comment:
cc @lttb

Reply:
hey! thanks for addressing it. I saw your latest change using `descriptionFileRoot`, and it makes a lot of sense but it seems that it might not be presented in some cases 🤔 I was investigating some similar issues recently and came up with this workaround, but it looks a bit clumsily, so I was hesitating to open a PR with these changes to be honest 😅

```ts
const mdxPath = context._module?.resourceResolveData
    ? // to make it work with symlinks, resolve the mdx path based on the relative path
      context._module.resourceResolveData.path ||
      context._module.resourceResolveData.descriptionFileRoot
      ? path.join(
          context._module.resourceResolveData.descriptionFileRoot,
          context._module.resourceResolveData.relativePath,
        )
      : path.join(
          context.rootContext,
          context._module.resourceResolveData.relativePath,
        )
    : context.resourcePath
```